src/index.js

```diff
@@ -35,14 +36,26 @@ function getAllAPIData() {
   })
 }
 
+function getCurrentDate() {
+  const unformattedDate = new Date;
+  const dateString = unformattedDate.toLocaleDateString("fr-CA", {
+    year: "numeric",
+    month: "2-digit",
+    day: "2-digit"
+  });
+  today = dateString.split('-').join('/');
+}
+
```

Comment:
I think here you are grabbing the current date and then formatting it to match the API data? I used this:

const date = new Date().toISOString().slice(0, 10).replaceAll('-', '/');
it takes the date, and turns it into ISO format (year/month/day), slices out only the date, and then replaces the "-" with "/"

and it could help DRY this function up.

Reply:
Yes I agree, however, I was actually trying that and the time zones got messed up later in the night, so I'm not sure what the best method is

Reply:
oh interesting, because this just slices the time zone out so it's not a factor, and just the date is.